src/Application.php

```diff
@@ -136,6 +136,23 @@ public function runCommandList($commandList, $options)
             $exitCode = $command->run($this->outputFile);
             if ($exitCode) {
                 return $exitCode;
+            } else {
+                /**
+                 * Make sure the project directory is removed also since `composer remove` will leave behind
+                 * an empty composer.json file.  This can potentially cause the `cgr info` command to abort
+                 * prior to completing the process of iterating through all the projects.
+                 */
+                list(, $workingDir, $composerCommand, $orgProject) = explode(' ', $command->getCommandString());
+                if ($composerCommand == "'remove'") {
```

Comment:
Could this be done in some other location, so that we do not need to decompose and parse the command string?